src/azure-cli/azure/cli/command_modules/appservice/appservice_environment.py

```diff
@@ -59,7 +58,8 @@ def create_appserviceenvironment_arm(cmd, resource_group_name, name, subnet, vne
 
     logger.info('Create App Service Environment...')
     deployment_name = _get_unique_deployment_name('cli_ase_deploy_')
-    ase_deployment_properties = _build_ase_deployment_properties(name, location, subnet_id, virtual_ip_type,
+    ase_deployment_properties = _build_ase_deployment_properties(cmd.cli_ctx, name, location,
```

Comment:
@madsd lets make sure to not use positional arguments so that we don't hit these sort of issues anymore

Reply:
After offline discussion with @panchagnula we decided to keep it this way.